include/acl.h

```diff
@@ -381,6 +381,8 @@ typedef struct {
 
     return result;
   }
+
```

Comment:
Maybe put it before the function so that it doesn't get missed when reading through the code?

Reply:
Thanks Sophie for reviewing. I think I will leave it as is for now. I tend to always add new data member to the existing struct at the end of struct for avoiding aggregate initialization of the struct somewhere else in the code base (not sure if it's a good/bad practice). Although in this case, put it before the function should be OK as it's still the end of the grouped data members. Since you don't seem to have strong opinion on this, I will leave it for now.